private_gpt/ui/ui.py

```diff
@@ -154,6 +157,41 @@ def build_history() -> list[ChatMessage]:
                     for index, source in enumerate(sources, start=1)
                 )
 
+    # On initialization and on mode change, this function set the system prompt
+    # to the default prompt based on the mode (and user settings).
+    @staticmethod
+    def _get_default_system_prompt(mode: str) -> str:
+        p = ""
+        match mode:
+            # For query chat mode, obtain default system prompt from settings
+            # TODO - Determine value to use if not defined in settings
+            case "Query Docs":
+                p = settings().local.default_query_system_prompt
+            # For chat mode, obtain default system prompt from settings or llama_utils
+            case "LLM Chat":
+                p = (
+                    settings().local.default_chat_system_prompt
+                    or llama_utils.DEFAULT_SYSTEM_PROMPT
```

Comment:
Let's not fall back to llama_index DEFAULT_SYSTEM_PROMPT. It is inconsistent with default_query_system_prompt (which doesn't have a default one). Also, we should not couple more than needed to llama_index; and the system should be able to work without a default_chat_system_prompt